vagrant.go

```diff
@@ -174,6 +187,20 @@ func (v *Vagrant) Setup(start bool, env string, numNodes int) error {
 	return nil
 }
 
+// Setup initializes a vagrant testbed.
+func (v *Vagrant) Setup(args ...interface{}) error {
```

Comment:
Let me think about it some more. We can always come back to this. Let's move forward for now.